rpm/kool-server.spec

```diff
@@ -44,17 +44,16 @@ mkdir -p %{buildroot}%{_exec_prefix}/dashboard
 
 %preun
 if [ $1 = 0 ]; then
-    # when the preun section is run, we've got stdin attached.  If we
-    # call stop() in the redis init script, it will pass stdin along to
-    # the redis-cli script; this will cause redis-cli to read an extraneous
-    # argument, and the redis-cli shutdown will fail due to the wrong number
-    # of arguments.  So we do this little bit of magic to reconnect stdin
-    # to the terminal
     term="/dev/$(ps -p$$ --no-heading | awk '{print $2}')"
     exec < $term
 
-    /sbin/service kool-server stop > /dev/null 2>&1 || :
+    /sbin/systemctl kool-server stop
```

Comment:
Systemctl is locate at ` /bin`
```
# which systemctl
/bin/systemctl
```
And you should pass the command as first argument and then the target:
```
/bin/systemctl stop kool-server.service
```